wp-content/themes/k1-theme/src/styles/pages/get-started.scss

```diff
@@ -0,0 +1,12 @@
+@use '../abstracts/mixins' as m;
+@use '../abstracts/variables' as v;
+
+
+.quotes {
+  background-color: $quotes-background-color;
```

Comment:
@reedar if this doesn't compile, it's because you need to prefix variables as `v.${variable-name}` (hence the import statement `@use '~/file' as v`